### ISSUE TITLE
Black screen/no boot after rebooting

### ISSUE THREAD

**Issue by crazyembereks** [ChimeraOS/chimeraos#913]
Hello, I am running alienware x51 R2 with a radeon rx 550. 
Install goes fine everything works well until I reboot and then chimeraos no longer boots for some reason

![issue](https://github.com/ChimeraOS/chimeraos/assets/94838821/7b5bea41-91dd-46c6-b52a-c92c7723b194)

**Comment by zongren** [comment]
same here
my pc is amd 7840u

**Comment by blankmindbm** [comment]
Heya,

I pieced together a workaround. Not sure if it's what is intended or not, but here goes:

-Power on
-Boot to black screen with underscore cursor
-Press CTRL+ALT+F2
-Login (Default is gamer/gamer)
-Switch the session to desktop using "chimera-session desktop". This should allow you to boot to the desktop to launch steam and then update it.
-After updating steam you may need to login again.
-Switch sessions again, but use "chimera-session steam"
-If going back to a blank black screen then switch the session to "chimera-session steam-plus"
-Allow the config to download and it should boot into game mode.

However, I was not able to use "Switch to Desktop" from the power menu in game mode. It actually froze my PC. I had to force my PC to shut down. Afterwards, I booted back up and followed the steps above, but this time instead of using "chimera-session steam-plus" I was able to use "chimera-session steam" and it booted into game mode. When trying "Switch to Desktop" my PC froze again. I forced it to power off again. This time when booting I was able to CTRL+ALT+F2 and the use "chimera-session steam" and it booted straight into game mode this time instead of having to first boot into the desktop. Instead of trying to switch to desktop mode I chose "Restart" this time. I still had to manually switch sessions to get the PC to boot to either Game Mode or Desktop.

In conclusion, you have to manually switch sessions each time you want to use either desktop or game mode and you have to select a session when booting the PC. Using the Game Mode icon will not switch to Game Mode and using "Switch to Desktop" will not switch to desktop. Maybe it could be a bad link in the UI? IDK, but this worked for me. Hoping an update will correct this soon! I would love to fully try ChimeraOS!


I'm running:
Dell OptiPlex 790
16GB RAM
i5-2400 x4
RX 550 4GB

**Comment by crazyembereks** [comment]
in another post I found a solution that worked for me. I had an HDD and that seems like what was causing the problem. I switched to an SSD and no longer have boot issues.

**Comment by pastaq** [comment]
We should add SSD as a minimum requirement for OS partition. HDD and SD/flash media are frequently failing

**Comment by ruineka** [comment]
I made some changes that may help booting in all cases. Can we all check unstable again? Thanks!
